### DISCUSSION TITLE
Entity Hider - Support Transparancy Percentage %

### DISCUSSION THREAD

**Discussion by dginovker** [runelite/runelite#16814] [Ideas]
This is a huge request, but...

For those who have played Realm of the Mad God, you know how many players can be in one area. It's important to see the other players, but in their full 3D color and outfits can be overbearing. 

Realm of the Mad God has a similar Entity Hider setting, but instead of being a Boolean toggle, it's a Slider where you control the transparency from 0 to 100%.

At 100%, you see everyone in full color, but at 0%, it's fully off. At 50%, you see the entities you've selected at 50% opacity.

**Comment by Endrit-alt** [comment]
Made the same request [here](https://github.com/runelite/runelite/discussions/16768). I don't think its gonna happen man :( Would be so cool to have.

![b424MVK](https://github.com/runelite/runelite/assets/69194598/f3e703f6-5431-474f-8d75-345cdc8e8cd6)
